token/program/src/lib.rs

```diff
@@ -1,5 +1,4 @@
 #![deny(missing_docs)]
-#![forbid(unsafe_code)]
```

Comment:
🙈 `SyscallStubs::sol_get_rent_sysvar()`, which is `cfg(test)` only requires an unsafe, and one can't override this directive later apparently.   Porting the SPL Token tests to use solana-program-test would avoid needing to make this change but that's too much work (tm)

Reply:
Does `#![cfg_attr(not(test), forbid(unsafe_code))]` work?